src/analysis/functions.rs

```diff
@@ -19,7 +19,7 @@ pub struct Info {
     pub glib_name: String,
     pub kind: library::FunctionKind,
     pub comented: bool,
-    pub class_name: Result,
+    pub owner_name: Result,
```

Comment:
`owner` would probably be clearer as `receiver` (as in receiver of the method) or `type`.